chaos-monkey-docs/src/main/asciidoc/changes.adoc

```diff
@@ -4,14 +4,13 @@
 Built with Spring Boot {spring-boot-version}
 
 === Bug Fixes
-- https://github.com/codecentric/chaos-monkey-spring-boot/pull/251[#251] Fix missing AssaultPropertiesUpdate validation.
+- https://github.com/codecentric/chaos-monkey-spring-boot/pull/252[#252] mvnw files update.
```

Comment:
Can you rephrase this into:  "Update Maven Wrapper from 0.5.5 to 0.5.6"

Reply:
As this is a build-only change, should it be in the changelog at all?

Reply:
It doesnt need to be, but it could. In the past we usually always added an entry (at least for the outside collaborators).